cdap-app-fabric/src/main/java/co/cask/cdap/internal/profile/ProfileService.java

```diff
@@ -75,6 +74,43 @@ public Profile getProfile(ProfileId profileId) throws NotFoundException {
     }, NotFoundException.class);
   }
 
+  /**
+   * Get the profile information about the given profile with property overrides. If a non-editable property is
+   * in the overrides, it will be ignored.
```

Comment:
Should we ignore the property? Then user will not be aware that the property will be overriden. Should we at least log a message about it?